.travis.yml

```diff
@@ -21,6 +21,12 @@ matrix:
     - php: hhvm
     - php: nightly
   fast_finish: true
+  include:
+    - name: "Guzzle 7 beta"
```

Comment:
For consistency and clarity's sake, could this be named something like `PHP: 7.3 - Guzzle 7 beta`